File: Proyectos_c/Ejercicio_09_Arreglos_Guardar y Mostrar.c
#include <stdio.h>

int main() {
    const int numerodedatos = 5;
    char nombres[numerodedatos][100];
    int enteros[numerodedatos];
    float floats[numerodedatos];

    for (int i = 0; i < numerodedatos; i++) {
        printf("Ingresa el nombre %d: ", i + 1);
        scanf("%s", nombres[i]);
    }

    for (int i = 0; i < numerodedatos; i++) {
        printf("Ingresa un número con decimales %d: ", i + 1);
        scanf("%f", &floats[i]);
    }

    for (int i = 0; i < numerodedatos; i++) {
        printf("Ingresa un número entero %d: ", i + 1);
        scanf("%d", &enteros[i]);
    }

    printf("Nombres ingresados\n");
    for (int i = 0; i < numerodedatos; i++) {
        printf("%s\n", nombres[i]);
    }

    printf("Números con decimales ingresados\n");
    for (int i = 0; i < numerodedatos; i++) {
        printf("%f\n", floats[i]);
    }

    printf("Números enteros ingresados\n");
    for (int i = 0; i < numerodedatos; i++) {
        printf("%d\n", enteros[i]);
    }

    return 0;
}

File: Proyectos_python/Ejercicio_09_Arreglos_Guardar y Mostrar.py
numerodedatos = 5
nombres = ["" for _ in range(numerodedatos)]
enteros = [0 for _ in range(numerodedatos)]
floats = [0.0 for _ in range(numerodedatos)]

for i in range(numerodedatos):
    nombres[i] = input("Ingresa el nombre {}: ".format(i + 1))

for i in range(numerodedatos):
    floats[i] = float(input("Ingresa un número con decimales {}: ".format(i + 1)))

for i in range(numerodedatos):
    enteros[i] = int(input("Ingresa un número entero {}: ".format(i + 1)))

print("Nombres ingresados:")
for nombre in nombres:
    print(nombre)

print("Números con decimales ingresados:")
for numero in floats:
    print(numero)

print("Números enteros ingresados:")
for numero in enteros:
    print(numero)